docs/distributions/manjaro/home.md

```diff
@@ -1,3 +1,7 @@
-Welcome to the mbp-manjaro wiki!
+# Deprecation Notice
+
+Manjaro support is deprecated. See [Deprecation Plan](https://wiki.t2linux.org/distributions/manjaro/deprecation) for more information.
```

Comment:
> Manjaro support is deprecated

Not yet from my understanding (this applies to the other pages with this note as well)